docs/src/main/asciidoc/spanner.adoc

```diff
@@ -25,51 +29,198 @@ dependencies {
 }
 ----
 
+This setup takes care of bringing in the latest compatible version of Cloud Java Spanner libraries as well.
+
+
+=== Configuration
+
+To setup Spring Data Spanner, you have to configure the following:
+
+* setup the connection details to Google Cloud Spanner
+* enable Spring Data Repositories (optional).
+
+==== Spanner settings
+
+You will need to use link:../spring-cloud-gcp-starters/spring-cloud-gcp-starter-data-spanner[Spring Boot Starter for Spring Data Spanner] to autoconfigure Google Cloud Spanner in your Spring application.
+It contains all the necessary setup that makes it easy to authenticate with your Google Cloud project.
+The following configuration options are available:
+
+|===
+| Name | Description | Optional | Default value
+| `spring.cloud.gcp.spanner.instance-id` | Spanner instance to use | No |
+| `spring.cloud.gcp.spanner.database` |
+Spanner database to use | No |
+| `spring.cloud.gcp.pubsub.project-id` | GCP project ID where the Google Cloud Pub/Sub API
```

Comment:
These still talk about pubsub instead of spanner

Reply:
ahh...copy paste, thank you.